src/pdn/doc/set_voltage_domain.md

```diff
@@ -0,0 +1,37 @@
+## set_voltage_domain
+
+### Synopsis
+```
+  % set_voltage_domain
+    -name name \
```

Comment:
`-grid` if you choose to change it

Reply:
In this case, name is the name of the voltage domain being defined - I corrected the description in the table